paw-util.el

```diff
@@ -140,8 +140,8 @@
   :group 'paw
   :type 'boolean)
 
-(defcustom paw-transalte-p t
-  "transalate automatically"
+(defcustom paw-translate-p t
```

Comment:
Don't forget to update the place where use it.

Reply:
I have reverted this for now. "transalte" is a common misspelling in the code base. I will make a more complete fix at a later time.